### DISCUSSION TITLE
The dependency "geographicslib" seems dead

### DISCUSSION THREAD

**Discussion by sscherfke** [geopy/geopy#580] [Ideas]
There haven't been any commits or releases for geographicslib since 2.5 years now: https://github.com/geographiclib/geographiclib-python

This should not be a problem _right now_ but might become one in the future.

**Comment by cffk** [comment]
No, it's not dead.  I have made small edits to the source code, mainly to keep the code base aligned with other implementations.  (And I've just pushed the corresponding commits.)  None have required a new release.
